app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] As clearly pointed out by the previous reviewer, please kindly add a belongs to relationship to the author as layed out in the (ERD Diagram)[https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png]